test/e2e-common.sh

```diff
@@ -484,9 +482,6 @@ function stage_serving_latest() {
   wget "${url}/serving-core.yaml" -P "${latest_dir}" \
     || fail_test "Unable to download latest knative/serving core file."
 
-  wget "${url}/serving-domainmapping.yaml" -P "${latest_dir}" \
-    || fail_test "Unable to download latest knative/serving domain mapping file."
-
```

Comment:
Leaving this would test upgrading. Do you want to skip that?

If no - we change this to not fail the script if it's a 404

Reply:
just seems easier to remove it since it'll always 404 from 7 days from now (I think?), so we're only getting 7 days of coverage here. Will happily add it, and then I guess remove it in a week or two, if you prefer we have the temporary coverage, though?

Reply:
Ok lets leave it as is